app/views/govuk_publishing_components/shared/_phase_banner.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="gem-c-phase-banner">
+  <p>
```

Comment:
Ideally wherever styles need to be applied we should be using an explicit class, but I appreciate that this CSS is being imported wholesale from somewhere else. It's probably okay, because the component's CSS is still namespaced to gem-c, but I think this should at least be recorded as technical debt.

Reply:
Yeah agree. Basically GOV.UK Frontend toolkit is future tech debt, once we introduce GOV.UK Frontend.